bor/README.md

```diff
@@ -0,0 +1,166 @@
+# PRELIMINARY TERMINOLOGY
```

Comment:
I'd prefer all lower case except first letter, but let's find a convention with @0xKrishna for all other modules

Reply:
I'm okay with either. But yeah, will wait for @0xKrishna to make the final call.

Reply:
I agree with @marcello33

Reply:
Done [here](https://github.com/maticnetwork/heimdall/pull/1071/commits/096779cf5bb71ba835d04dd5fa866515ed045a6a).